pages/common/pkgz.md

```diff
@@ -0,0 +1,24 @@
+# pkgz
+
+> Unified package manager for installing, removing, updating, and searching packages from multiple sources.
+> More information: <https://github.com/roguehashrate/pkgz#-usage>
```

Comment:
```suggestion
> More information: <https://github.com/roguehashrate/pkgz#-usage>.
```
Odd. The bot should be complaining about the missing dot and not an extra empty line. @sebastiaanspeck